build/ci/tools/osuosl/publish.sh

```diff
@@ -54,16 +58,20 @@ chmod 600 $SSH_KEY
 
 FTP_PATH=${OS}/${MAJOR_VERSION}x/${BUILD_DIR}
 
+file_extension="${ARTIFACT_NAME##*.}"
+LATEST_NAME="MuseScoreNightly-latest-x86_64.${file_extension}"
+
 echo "Copy ${ARTIFACTS_DIR}/${ARTIFACT_NAME} to $FTP_PATH"
 scp -oStrictHostKeyChecking=no -C -i $SSH_KEY $ARTIFACTS_DIR/$ARTIFACT_NAME musescore-nightlies@ftp-osl.osuosl.org:~/ftp/$FTP_PATH
 
 # For Linux, we also need to send a .zsync file, if exists
-ZSYNC_EXISTS=0
-if [ "$OS" == "linux" ]; then 
+if [ "$OS" == "linux" ]; then
     if [ -f "$ARTIFACTS_DIR/${ARTIFACT_NAME}.zsync" ]; then
         echo "Copy ${ARTIFACTS_DIR}/${ARTIFACT_NAME}.zsync to $FTP_PATH"
+        sleep 1m # For fine sort order on FTP
```

Comment:
I looked, it is still not sorted correctly, all 4 files have the same time. I think I need to try to set 2 minutes.
If that doesn't work, we might need to call the trigger, then delay, create links, and call the trigger again.